rj_gameplay/classifier/pass_success_classifier.py

```diff
@@ -0,0 +1,95 @@
+import stp.rc as rc
+import rj_gameplay.gameplay_node as gameplay_node
+import rclpy
+
+import numpy as np
+import math as m
+
+def classifier(world_state: rc.WorldState) -> bool:
+    our_robots = world_state.our_robots
+    opp_robots = world_state.their_robots
+    ball = world_state.ball
+
+    # check if any robot on our team has the ball 
+    for robot in our_robots:
+        if robot.has_ball:
+            return True
+
+    for robot in opp_robots:
+        if robot.has_ball:
+            return False
+
+    net_robot_list = our_robots + opp_robots
+
+    robot = shortest_distance(rc.WorldState, net_robot_list, ball)
+
+    if robot == None:
+        return False
+
+    if robot in our_robots:
+        return True
+    
+    return False
+    
+
+def shortest_distance(world_state: rc.WorldState, robot_list, ball) -> rc.Robot:
+    ball_x = ball.pos()[0]
+    ball_y = ball.pos()[1]
+
+    ball_vel_x = ball.vel()[0]
+    ball_vel_y = ball.vel()[1]
+
+    minDistance = 100000
```

Comment:
Another nitpick, but instead of using a very large number use float('inf') which will basically be infinity